cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -29,10 +31,19 @@
                         at the bottom of the molecule.
 
    ========================================================================== #}
-
+{% if value.eyebrow and value.heading %}
```

Comment:
Should this be `{% if value.eyebrow or value.heading %}`?

Reply:
I think it could be just `if value.eyebrow`, since the validation ensures that if an eyebrow is present, a heading is required.